pywrap.bzl

```diff
@@ -165,6 +172,79 @@ def pybind_extension(
         deps = ["%s" % cc_library_name],
     )
 
+def _generated_pywrap_win_def_file_impl(ctx):
+    cc_toolchain = find_cpp_toolchain(ctx)
+    feature_configuration = cc_common.configure_features(
```

Comment:
This was needed to do lib_copy, sinc eyou do not need to do libcopy here you don't need this part as well.